<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericTelemetryDialog</class>
 <widget class="QDialog" name="GenericTelemetryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telemetry Page</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>730</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;*No packets? Remember to select the IP address of your&lt;br/&gt;spacecraft in the Main Window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblTelemetry">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>470</width>
     <height>601</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>225</number>
   </attribute>
   <column>
    <property name="text">
     <string>Telemetry Point Label</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telemetry Point Value</string>
    </property>
   </column>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>25</y>
     <width>73</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="packetIdLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Packet ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="packetId">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>23</y>
     <width>125</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Sequence Count</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sequenceCount">
      <property name="minimumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>16384</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>24</y>
     <width>204</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="subSystemTelemetryPageLabel">
      <property name="text">
       <string>Subsystem Telemetry Page</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subSystemLineEdit">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>31</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>GenericTelemetryDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
